### PR TITLE
Make mobile icons default to 22 if no sizeType specified

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers 
@cecileboucheron

### CODE REVIEW

File: shared/common-adapters/icon.native.js [hunk 1]
```diff
@@ -118,8 +118,7 @@ class Icon extends React.PureComponent<Props> {
       } else if (this.props.sizeType) {
         fontSize = Shared.typeToFontSize(this.props.sizeType)
```
Comment (cecileboucheron) [inline shared/common-adapters/icon.native.js]:
```suggestion
} else {
        fontSize = Shared.typeToFontSize(this.props.sizeType || 'Default')
```

**Comment by cecileboucheron** [comment]
👍🏻

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/common-adapters/icon.desktop.js [hunk 2]
```diff
@@ -52,8 +55,6 @@ class Icon extends Component<Props, void> {
       fontSizeHint = {fontSize: this.props.fontSize}
     } else if (this.props.sizeType) {
```
Comment (buoyad) [inline shared/common-adapters/icon.desktop.js]: Just `else`

**Comment by chrisnojima** [comment]
@cecileboucheron i put a sizeType tiny on all caret-right / down

**Pull request merged**